apps/src/applab/export.js

```diff
@@ -0,0 +1,7 @@
+import * as api from './api';
+import {executors} from '../lib/util/audioApi';
+import * as dontMarshalApi from './dontMarshalApi';
+
+export function getExportedGlobals() {
+  return Object.assign({}, executors, dontMarshalApi, api);
```

Comment:
:+1: To isolating this weirdness into its own file.  It'd be nice to have a comment explaining what this is for though - it's not obvious to me in isolation.